cswinrt/strings/additions/Windows.Storage.Streams/IBufferByteAccess.cs

```diff
@@ -13,8 +13,10 @@ internal interface IBufferByteAccess
 namespace ABI.Windows.Storage.Streams
 {
     using global::System;
-    using global::System.Runtime.InteropServices;
-
+    using global::System.Runtime.InteropServices;
+//TODO: Update netstandard version by adding objectreferenceattribute 

```

Comment:
is this a todo for this PR or in the future? if future, can we open an issue for it?

Reply:
if possible, let's address it in this PR while you have context, etc

---
In reply to: [506975159](https://github.com/microsoft/CsWinRT/pull/512#discussion_r506975159) [](ancestors = 506975159)